Predecessor Gas Giant PR (#11313)
endless-sky/endless-sky: data/successors/successor side missions.txt

					goto what
				`	"You are afraid."`
			`	"Aye, and thou should'st be as well." The Successor turns away, unwilling to speak further.`
				decline
			label what
			`	"By that which should never wake again." The Successor's voice drops lower. "Leave it. Let the storm take it. Naught but ruin shall come of this."`
				decline

mission "Successors: Gas Station 2 House Seineq"
	minor
	landing
	invisible
	source
		government "House Seineq"
	to offer
		has "Successors: Gas Station 1: offered"
		has "known to the successors"
	to fail
		has "Successors: Gas Station 1: done"
	on offer
		conversation
			branch alreadyasked
				and
					not "trusted by the successors"
					has "Successors: asked about the gas station"
			`Do you want to ask House Seineq again about the locked station hidden in the gaseous atmosphere of Avasaa-Aa-Tej, now that you earned some of their trust?`
				to display
					has "Successors: asked about the gas station"
				goto newquestion
			`Do you want to ask House Seineq about the locked station hidden in the gaseous atmosphere of Avasaa-Aa-Tej?`
				to display
					not "Successors: asked about the gas station"
				goto newquestion

			label alreadyasked

> 			`You try once more to press a scion of House Seineq about the station at Avasaa-Aa-Tej, but they remain impassive.`

Reviewer: You're missing the place where you set the `"Successors: asked about the gas station"` condition.
```suggestion
			action
				set "Successors: asked about the gas station"
			`You try once more to press a scion of House Seineq about the station at Avasaa-Aa-Tej, but they remain impassive.`
```

			choice
				`	"What sort of device?"`
					goto explain
				`	"Why do you need me to test this device?"`
					goto convince
				`	"I'm not interested."`
					decline
			label explain
			`	"It is a new drive, meant not to step through the void, but to hurl oneself into it unburdened by stillness. Unlike our lesser designs, it does not demand stillness before the leap."`
			`	They hesitate, chromatophores dimming momentarily. "Yet, there are flaws. The flow of energy within is.. erratic. Its grasp upon the unseen currents is incomplete. Thou art no stranger to the void's pathing-perhaps thy hands may guide its course where ours have failed."`
			choice
				`	"What do you need me to do?"`
					goto task
				`	"I'm not risking my ship for an experiment."`
					decline
			label convince
			`	Taaquvaleq's chromatophores ripple in a slow, deliberate grayish wave. "We do not ask lightly. But thou dost wander freely across the great dark. What might a swifter path be worth to thee?"`
			choice
				`	"Fine, I'll do it."`
					goto task
				`	Sorry, but you should find someone else."`
					decline
			label task

> 			`	"Take this drive and set it within thy vessel. Then, venture to the system of Myruet-Kvelq, where the storms of hyperspace rage most fiercely. If the drive holds, then we shall have achieved a feat most rare. If it fails.." `

Reviewer: ```suggestion
			`	"Take this drive and set it within thy vessel. Then, venture to the system of Myruet-Kvelq, where the storms of hyperspace rage most fiercely. If the drive holds, then we shall have achieved a feat most rare. If it fails..."`
```

> 			`	Taaquvaleq pauses, their chromatophores flickering with tension. "Then thou must learn why, and return with the knowing."`

Reviewer: ```suggestion
			`	Taaquvaleq pauses, their chromatophores flickering with tension. "Then thou must learn why, and return with the knowledge."`
```

				accept
	on accept
		outfit "Prototype Quick Drive"
	
	on enter "Myruet-Kvelq"
		require "Prototype Quick Drive"
		conversation
			`The prototype drive hums beneath your feet, its presence thrumming through the ship's frame.`
			`	As you engage it at speed, the stars ripple-far more than a normal hyperspace jump should allow. A sudden jolt wracks the ship, and the void itself seems to distort unnaturally.`
			choice
				`	(Attempt to stabilize the drive.)`
					goto stabilize
				`	(Abort the jump and return.)`
					goto return
			label stabilize
			`	With careful adjustments, you manage to counter the drive's erratic oscillations, completing the jump-barely. Your ship's energy reserves, however, drain faster than expected.`
				goto report
			label return
			`	Disengaging the drive, you retreat to normal space and prepare to return to House Seineq. The drive is flawed. Taaquvaleq will want the data to learn more.`
			label report
	on complete
		outfit "Prototype Quick Drive" -1
		log "Tested House Seineq's experimental Quick Drive. Results indicate instability caused by high energy usage."
		conversation

> 			`	Taaquvaleq watches intently as you relay the results. "Hmm. The instability lingers, and the fuel draw.. excessive. The weave of space does not yet heed our call as it once did in the age of our Predecessors."`

Reviewer: ```suggestion
			`Taaquvaleq watches intently as you relay the results. "Hmm. The instability lingers, and the fuel draw... excessive. The weave of space does not yet heed our call as it once did in the age of our Predecessors."`
```

> 			`	They tilt their head slightly, considering. "Perhaps our failing is not in the craft, but in the materials. Our Predecessors drew power from sources we no longer wield. If such remnants still linger, they may grant us clarity." Their gaze fixes upon you once more. "But there is no place such relics may yet be found..."`

Reviewer: ```suggestion
			`	They tilt their head slightly, considering. "Perhaps our failing is not in the craft, but in the materials. Our Predecessors drew power from sources we no longer wield. If such remnants still linger, they may grant us clarity." Their gaze fixes upon you once more. "But there is no place such relics may yet be found...`
```

			`	"Return to the spaceport when the knowing is made whole, and the echoes of the logs have stilled."`
			
mission "Successors: Gas Station 4"
	name "A Bold Proposal"

> 	description `Travel with Taaquvaleq to Yoqqa-Vasa-Vasa to negotiate with House Seineq about the station at Avasaa-Aa-Tej`

Reviewer: ```suggestion
	description `Travel with Taaquvaleq to <planet> to negotiate with House Seineq about the station in Avasaa-Aa-Tej.`
```

	landing
	source
		government "New Houses"
	destination "Yoqqa-Vasa-Vasa"
	passengers 1
	to offer
		has "Successors: Gas Station 3: done"
	on offer
		conversation

> 			`Taaquvaleq meets you again, their chromatophores a deep, unwavering violet. "We have studied the drive's failure, and in its core, we see a shadow of the past."`
> 			`Our Predecessors used special isotopes not as fuel, but as catalysts - their molecular lattice absorbing hyperspace resonance, allowing their Drives to 'breathe' without tearing itself apart. Our prototype lacks this... refinement."`
> 			`"These isotopes that once served our Predecessors' Drives - those may be the key to perfecting our own design." A ripple of deep blue, signaling sorrow, courses over their skin. "Yet these are lost to the storms of Avasaa-Aa-Tej."`

Reviewer: ```suggestion
			`Taaquvaleq meets you again, their chromatophores a deep, unwavering violet. "We have studied the drive's failure, and in its core, we see a shadow of the past.`
			`	"Our Predecessors used special isotopes not as fuel, but as catalysts - their molecular lattice absorbing hyperspace resonance, allowing their Drives to 'breathe' without tearing itself apart. Our prototype lacks this... refinement.`
			`	"These isotopes that once served our Predecessors' drives - those may be the key to perfecting our own design." A ripple of deep blue, signaling sorrow, courses over their skin. "Yet these are lost to the storms of Avasaa-Aa-Tej."`
```

			choice
				`	(Tell Taaquvaleq about the refinery you discovered in the atmosphere of Avasaa-Aa-Tej owned by House Seineq.)`

> 			`You recount your journey through the storm of Avasaa-Aa-Tej, how within the swirling haze, a locked station endured, bearing the claim of House Seineq. At the mention, Taaquvaleq's chromatophores flicker-first amber, then an uneasy shade of blue.`
> 			`"Avasaa-Aa-Tej? That place holds naught but ruin and shadow. Yet.. thou say'st a refinery remains?" They fall silent, contemplation washing over them like a slow tide.`
> 			`Finally, they exhale. "If what thou claim'st is true, then that which was lost may yet be reclaimed. The refinery that lingers there, sealed and forgotten, may still hold remnants of its old yield. If thou would'st recover these, we may yet tame the wild heart of our device."`

Reviewer: ```suggestion
			`	You recount your journey through the storm of Avasaa-Aa-Tej, how within the swirling haze, a locked station endured, bearing the claim of House Seineq. At the mention, Taaquvaleq's chromatophores flicker - first amber, then an uneasy shade of blue.`
			`	"Avasaa-Aa-Tej? That place holds naught but ruin and shadow. Yet... thou say'st a refinery remains?" They fall silent, contemplation washing over them like a slow tide.`
			`	Finally, they exhale. "If what thou claim'st is true, then that which was lost may yet be reclaimed. The refinery that lingers there, sealed and forgotten, may still hold remnants of its old yield. If thou would'st recover these, we may yet tame the wild heart of our device."`
```

			choice
				`	"Then let us reclaim it."`
					goto requiremeeting
				`	"I cannot act alone in this; we must gain the access codes of House Seineq first."`
					goto proposemeeting
				`	"Perhaps this should remain buried."`
					decline
			label requiremeeting

> 			`	Taaquvaleq tilts their head slightly. "A noble thought, yet Seineq doth not act on whispers alone. The scions of thy House must weigh what is known, and judge to grant us access. Let us depart for Yoqqa-Vasa-Vasa."`
> 				accept
> 			label proposemeeting
> 			`	"A meeting, then, with those whose will doth shape our course. Let us depart for Yoqqa-Vasa-Vasa."`

Reviewer: ```suggestion
			`	Taaquvaleq tilts their head slightly. "A noble thought, yet Seineq doth not act on whispers alone. The scions of thy House must weigh what is known, and judge to grant us access. Let us depart for <planet>."`
				accept
			label proposemeeting
			`	"A meeting, then, with those whose will doth shape our course. Let us depart for <planet>."`
```

				accept
				
	on complete
		conversation

> 			`You request for an audience with Seineq Veythiq, the very Scion of House Seineq who first spoke to you of the refinery at Avasaa-Aa-Tej, which remains under their House's claim.`

Reviewer: ```suggestion
			`You request an audience with Seineq Veythiq, the Scion of House Seineq who first spoke to you of the refinery at Avasaa-Aa-Tej, which remains under their House's claim.`
```

> 			`	He stands waiting as you arrive, esides them, towers an armored guard, who watches you with careful scrutiny.`

Reviewer: ```suggestion
			`	They stand waiting as you arrive. Beside them towers an armored guard, who watches you with careful scrutiny.`
```

			`	"Thou return'st with an offering, I presume? Speak then, and swiftly."`
			choice

> 				`(Let Taaquvaleq speak.)`
> 				`"This engineer has a proposal for you."`
> 
> 			`Taaquvaleq steps forward, bows low, their voice even. "House Seineq holds claim over the station's remnants at Avasaa-Aa-Tej. Thy House I seek what little remains - and in return, I shall share what I have built."`

Reviewer: ```suggestion
				`	(Let Taaquvaleq speak.)`
				`	"This engineer has a proposal for you."`

			`	Taaquvaleq steps forward and bows low, their voice even. "House Seineq holds claim over the station's remnants at Avasaa-Aa-Tej. Thy House I seek what little remains - and in return, I shall share what I have built."`
```

> 			`	"A bold trade, yet an unbalanced one. Thou would'st offer scraps of wisdom in exchange for what is ours by right? The station, the yield within-House Seineq hath laid claim since time immemorial.`

Reviewer: ```suggestion
			`	"A bold trade, yet an unbalanced one. Thou would'st offer scraps of wisdom in exchange for what is ours by right? To the station, the yield within, House Seineq hath laid claim since time immemorial.`
```

			`	"Yet..." Seineq Veythiq's skin ripples with reluctant violet, their arms coiling in contemplation. "None but thee have braved the storm's heart and returned to speak of it. Our own ships falter where thine endures." A pause, their chromatophores flickering with grudging amber acknowledgment. "If any hand may reclaim what was lost, 'tis thine." The guard shifts uneasily, their metallic bands clinking softly as the scion continues: "Thus we grant thee passage - but mark this well: what is retrieved comes first to Seineq's vaults. Prove thy worth in this, and then... perhaps some portion may be allotted to thy engineer's ambitions."`

> 			`The guard steps forward, extending a data-sphere that glows faint blue. "These codes grant passage," they rumble. "But know this - I shall accompany thee not as escort, but as warden. What is retrieved comes first to House Seineq. Only then shall we consider its disposition."`
> 			`Taaquvaleq's skin flickers yellow-green - nervous hope. "The prototype requires but a fraction of the isotopes. Surely-"`
> 			`The scion raises a limb, cutting them off. "No bargains ere the property is secured. The guard shall record all findings. Once the haul is weighed and catalogued in our vaults... then we may speak of shares."`
> 			`Their chromatophores darken to a warning indigo. "Attempt to withhold even a grain, and not only thy prototype shall be forfeit, but thy right to walk our halls."`

Reviewer: ```suggestion
			`	The guard steps forward, extending a data-sphere that glows faint blue. "These codes grant passage," they rumble. "But know this - I shall accompany thee not as escort, but as warden. What is retrieved comes first to House Seineq. Only then shall we consider its disposition."`
			`Taaquvaleq's skin flickers yellow-green - nervous hope. "The prototype requires but a fraction of the isotopes. Surely-"`
			`	The scion raises a limb, cutting them off. "No bargains ere the property is secured. The guard shall record all findings. Once the haul is weighed and catalogued in our vaults... then we may speak of shares."`
			`	Their chromatophores darken to a warning indigo. "Attempt to withhold even a grain, and not only thy prototype shall be forfeit, but thy right to walk our halls."`
```

mission "Successors: Gas Station 5"
	name "The Ancient Refinery"

> 	description `Bring Taaquvaleq and the Seineq Guard to the Station at Avasaa-Aa-Tej.`

Reviewer: ```suggestion
	description `Bring Taaquvaleq and the Seineq guard to the station in <planet>.`
```

	landing
	source
		government "House Seineq"
	destination "Avasaa-Aa-Tej"
	passengers 2
	to offer
		has "Successors: Gas Station 4: done"
	on offer

> 		dialog "Taaquvaleq and the Seineq Guard board your ship, you should depart for Avasaa-Aa-Tej."

Reviewer: ```suggestion
		dialog "Taaquvaleq and the Seineq Guard board your ship. You should depart for Avasaa-Aa-Tej."
```

		event "Successors: Gas Station add Spaceport"
	npc
		government "Unknown House"
		personality secretive staying
		ship "Aaulqra (Sniper)" "Eye of the Storm"
		system "Myruet-Kvelq"
	on complete
		conversation

> 			`As your ship plunges deeper into the swirling storms of Avasaa-Aa-Tej, the dense, atmosphere presses against your hull. Visibility drops to near zero, your sensors struggling to cut through the haze.`

Reviewer: ```suggestion
			`As your ship plunges deeper into the swirling storms of Avasaa-Aa-Tej the dense, atmosphere presses against your hull. Visibility drops to near zero, your sensors struggling to cut through the haze.`
```

			`	Through the distortion, the station finally emerges-a monolithic structure, its metal frame battered by centuries of relentless winds yet standing defiantly against the storm's fury. The automated transmission begins again, repeating the same cryptic identification request you first encountered.`

> 			`	The Seineq Guard steps forward, their voice calm but firm. "Transmit the codes. Let us see if time hath eroded their meaning."`

Reviewer: ```suggestion
			`	The Seineq guard steps forward, their voice calm but firm. "Transmit the codes. Let us see if time hath eroded their meaning."`
```

			`	Taaquvaleq fidgets slightly, their chromatophores pulsing a nervous green. "If these do not work, I fear we may need a... less elegant solution."`

> 			`With a quick input, you send the access codes provided by House Seineq. For a long, tense moment, nothing happens. Then, with a deep groan of ancient machinery, the station stirs. Metal locks disengage with a heavy clunk, and an exterior platform extends from the side of the structure, illuminated by flickering red emergency lights. A docking corridor hisses open, revealing a passageway into the refinery's depths.`

Reviewer: ```suggestion
			`	With a quick input, you send the access codes provided by House Seineq. For a long, tense moment, nothing happens. Then, with a deep groan of ancient machinery, the station stirs. Metal locks disengage with a heavy clunk, and an exterior platform extends from the side of the structure, illuminated by flickering red emergency lights. A docking corridor hisses open, revealing a passageway into the refinery's depths.`
```

			`	Stepping inside, the air is thick, stale, and laced with the scent of industrial decay. The walls are covered in layers of frost where life support systems have long since faltered. Dim emergency lighting provides just enough illumination to reveal the vast corridors-hallways built for purposes long forgotten, their surfaces worn by time.`
			`	As you explore, you find evidence of ancient machinery, massive processing units still wired into the refinery's core. But they are dead. Silent. Whatever fueled this place has long since run dry.`

> 			`	Taaquvaleq moves between the consoles, running a hand over the corroded interfaces. "A relic," they murmur, their chromatophores shifting into thoughtful blues. "It breathes no more, yet... it remembers.`
> 			`	The Seineq Guard, standing alert, clicks their armored limbs together. "We have no time for reverie. We came for the yield."`
> 			`Further in, you find rows of massive storage containers, their identification markings faded. Most are empty-some completely hollowed out, others lined with only the faintest traces of residual isotopes. But at the very end of the chamber, one container remains full.`

Reviewer: ```suggestion
			`	Taaquvaleq moves between the consoles, running a hand over the corroded interfaces. "A relic," they murmur, their chromatophores shifting into thoughtful blues. "It breathes no more, yet... it remembers."`
			`	The Seineq guard, standing alert, clicks their armored limbs together. "We have no time for reverie. We came for the yield."`
			`	Further in, you find rows of massive storage containers, their identification markings faded. Most are empty - some completely hollowed out, others lined with only the faintest traces of residual isotopes. But at the very end of the chamber, one container remains full.`
```

			`	As you scan it, your instruments confirm its contents: a large cache of refined isotopes, preserved through centuries, waiting. The very material Seineq once used in their greatest technological ambitions.`
			`	Taaquvaleq's chromatophores flare with sudden excitement. "This... this is enough! A fraction of this could restore what was lost! We-"`

> 			`	The Guard steps between them and the container, their skin shifting to an authoritative gray. "It shall go to Seineq's vaults first. All of it."`
> 			`	Taaquvaleq's excitement dims, their chromatophores flickering uncertainly, but they hold their tongue. For now.`
> 			`You make preparations to transport the haul.`

Reviewer: ```suggestion
			`	The guard steps between them and the container, their skin shifting to an authoritative gray. "It shall go to Seineq's vaults first. All of it."`
			`	Taaquvaleq's excitement dims, their chromatophores flickering uncertainly, but they hold their tongue. For now.`
			`	You make preparations to transport the haul.`
```

		
mission "Successors: Gas Station 6"

> 	name "A Cargo load full of treasures"

Reviewer: ```suggestion
	name "A cargo of treasures"
```
I think the name's too long otherwise.

> 	description `Bring Taaquvaleq and the Seineq Guard and the Isotopes from Avasaa-Aa-Tej back to Yoqqa-Vasa-Vasa.`

Reviewer: ```suggestion
	description `Bring Taaquvaleq, the Seineq guard, and the isotopes from <origin> back to <destination>.`
```

	landing
	source "Avasaa-Aa-Tej"
	destination "Yoqqa-Vasa-Vasa"
	passengers 2
	cargo "Predecessor Isotopes" 190

> 	blocked "You will need at least 190t of free cargo space and two free bunks to continue."

Reviewer: This is.... a lot. Perhaps add 190 tons of `reserved` cargo to the previous mission, just in case someone gets stuck here?

	to offer
		has "Successors: Gas Station 5: done"
	on offer

> 		dialog "You use the ancient loading mechanics to load the Isotopes cache onto your ship and depart together with Taaquvaleq and the Seineq Guard to deliver House Seineq their property."

Reviewer: ```suggestion
		dialog "You use the ancient loading mechanics to load the isotope cache onto your ship and depart together with Taaquvaleq and the Seineq guard to deliver House Seineq their property."
```

		"reputation: Unknown House" = -100
	npc
		government "Unknown House"
		personality nemesis
		ship "Aaulqra (Sniper)" "Eye of the Storm"
		system "Myruet-Kvelq"
	on complete
		conversation

> 			`As you land at the grand estate of House Seineq, the station's atmosphere feels tense yet expectant. The Seineq Guard disembarks first, striding ahead to announce your arrival. Taaquvaleq follows, their chromatophores flickering between anxious blues and cautious greens.`
> 			`Once more, you are led through ornate halls, their towering columns shimmering under the glow of bioluminescent engravings. At the far end of the chamber, the Seineq Veythiq awaits, adorned in elaborate metallic bands that signal both power and tradition.`
> 			`The Scion of House Seineq extends a limb, their chromatophores shifting to deep violet as they study the recovered isotopes. "Thou return'st, and with that which was promised." The words carry the weight of solemn approval. "A task well-wrought, and well-received."`
> 			`Their gaze drifts to Taaquvaleq, whose skin flashes bright green-elation-before settling into respectful indigo. "House Seineq doth honor its word. Thee shall have two weight of the isotopes recovered-no more, no less." Veythiq's voice turns precise, each syllable measured. "'Tis enough to complete thy craft, yet scarce enough to tempt folly."`
> 			`"Thy House is most generous," Taaquvaleq murmurs, their limbs curling in a gesture of gratitude. "With but this fraction, I may weave anew what was once lost."`
> 			`A sudden tension fills the chamber as Seineq Veythiq steps forward, their chromatophores darkening like storm clouds. "Now, mark this well, travelers." The air grows heavy with unspoken threat. "What thou hast seen, what thou hast taken-none but thee shall know. House Seineq grants thee this yield, but in return, we demand silence."`
> 			`Behind them, the armored Seineq guard moves with deliberate menace, their segmented plates scraping against each other. "Swear it now," they rumble, the sound vibrating through the damp air, "before the House, before the void itself. No record, no tale, no whispered word beyond these halls."`
> 			`The Scion's skin ripples black at the edges-a final warning. "The secrets of Avasaa-Aa-Tej are not for the tongues of lesser minds. Speak naught of what transpired, nor of what was found..." They let the pause stretch, their meaning clear. "...lest thou invite ruin upon thine own house."`
> 			`As you prepare to depart, Taaquvaleq draws near, "If thy curiosity yet lingers, traveler... meet me upon O-Vasa-Oa. The final threads are almost woven. None else shall test it. If thou wouldst see what hath become of our labor... come."`

Reviewer: ```suggestion
			`As you land at the grand estate of House Seineq, the station's atmosphere feels tense yet expectant. The Seineq guard disembarks first, striding ahead to announce your arrival. Taaquvaleq follows, their chromatophores flickering between anxious blues and cautious greens.`
			`	Once more, you are led through ornate halls, their towering columns shimmering under the glow of bioluminescent engravings. At the far end of the chamber, the Seineq Veythiq awaits, adorned in elaborate metallic bands that signal both power and tradition.`
			`	The Scion of House Seineq extends a limb, their chromatophores shifting to deep violet as they study the recovered isotopes. "Thou return'st, and with that which was promised." The words carry the weight of solemn approval. "A task well-wrought, and well-received."`
			`	Their gaze drifts to Taaquvaleq, whose skin flashes bright green - elation - before settling into respectful indigo. "House Seineq doth honor its word. Thee shall have two weight of the isotopes recovered - no more, no less." Veythiq's voice turns precise, each syllable measured. "'Tis enough to complete thy craft, yet scarce enough to tempt folly."`
			`	"Thy House is most generous," Taaquvaleq murmurs, their limbs curling in a gesture of gratitude. "With but this fraction, I may weave anew what was once lost."`
			`	A sudden tension fills the chamber as Seineq Veythiq steps forward, their chromatophores darkening like storm clouds. "Now, mark this well, travelers." The air grows heavy with unspoken threat. "What thou hast seen, what thou hast taken - none but thee shall know. House Seineq grants thee this yield, but in return, we demand silence."`
			`	Behind them, the armored Seineq guard moves with deliberate menace, their segmented plates scraping against each other. "Swear it now," they rumble, the sound vibrating through the damp air, "before the House, before the void itself. No record, no tale, no whispered word beyond these halls."`
			`	The Scion's skin ripples black at the edges - a final warning. "The secrets of Avasaa-Aa-Tej are not for the tongues of lesser minds. Speak naught of what transpired, nor of what was found..." They let the pause stretch, their meaning clear. "...lest thou invite ruin upon thine own house."`
			`	As you prepare to depart, Taaquvaleq draws near, "If thy curiosity yet lingers, traveler, meet me upon O-Vasa-Oa. The final threads are almost woven. None else shall test it. If thou wouldst see what hath become of our labor, come."`
```

mission "Successors: Gas Station 7A"
	name "Meet Taaquvaleq at his workshop"
	description `Travel to O-Vasa-Oa to help test the perfected Quick Drive.`
	destination "O-Vasa-Oa"
	to offer
		has "Successors: Gas Station 6: done"
		not "Successors: Gas Station 7: offered"

> 	on offer
> 		conversation
> 			`As you prepare to depart, Taaquvaleq draws near, "If thy curiosity yet lingers, traveler... meet me upon O-Vasa-Oa. The final threads are almost woven. None else shall test it. If thou wouldst see what hath become of our labor... come."`
> 				accept

Reviewer: They already say this above.
```suggestion
```

		
mission "Successors: Gas Station 7"
	name "Test the refined Quick Drive"

> 	description `Install the refined Quick Drive and test fly it again to Uuoru-Veldt-Stir in the Myruet-Kvelq System, then return your findings to Taaquvaleq at O-Vasa-Oa.`

Reviewer: ```suggestion
	description `Install the refined Quick Drive and test fly it again to <stopovers>, then return your findings to Taaquvaleq on <planet>.`
```

	landing
	stopover "Uuoru-Veldt-Stir"
	destination "O-Vasa-Oa"
	source
		government "New Houses"
	to offer
		has "Successors: Gas Station 6: done"
	npc
		government "Unknown House"
		personality nemesis staying
		fleet
			names "successor"
			variant
				"Aaulqra" 3
				"Veusade" 2
				"Seiitej" 2
		system "Myruet-Kvelq"
	on offer
		conversation

> 			`	Taaquvaleq greets you with a steady gaze, their chromatophores a deep, confident violet.`

Reviewer: ```suggestion
			`Taaquvaleq greets you with a steady gaze, their chromatophores a deep, confident violet.`
```

			`	"'Tis complete," they say simply. "The drive thou hast tested-flawed, crude, incomplete-now stands whole. The isotopes from Avasaa-Aa-Tej have woven stability into the weave, binding it as once was done in the age before."`
			`	They gesture to a sealed container, within which the refined Quick Drive hums with controlled power. "Yet knowing and truth are not the same. One last proving remains."`
			choice
				`	"What do you need me to do?"`
					goto task
				`	"Why not test it yourself?"`

> 					goto convince

Reviewer: ```suggestion
```

				`	"I'm not interested."`
					decline
			label convince
			`	Taaquvaleq's chromatophores shift with a ripple of mirth. "We craft, we refine, but 'tis thee who walks the void as we cannot. Thy vessel is suited to this task. And if all is well, the drive shall be thine to keep."`
			choice
				`	"Fine. Where do I take it?"`
					goto task

> 				`	Sorry, but I am not interested anymore in this project."`

Reviewer: ```suggestion
				`	"Sorry, but I am not interested in this project anymore."`
```

					decline
			label task

> 			`	"Install the drive and test fly it to Uuoru-Veldt-Stir in the Myruet-Kvelq System, where the borders of hyperspace fray. There, at the precipice of the void, engage the drive at full velocity. If it holds, then our work is complete."`

Reviewer: ```suggestion
			`	"Install the drive and test fly it to <stopovers>, where the borders of hyperspace fray. There, at the precipice of the void, engage the drive at full velocity. If it holds, then our work is complete."`
```

			`	They incline their head slightly. "Thou need'st only return with the knowing. This final test is thine alone."`
				accept
	on accept
		outfit "Quick Drive"
	on enter "Myruet-Kvelq"

> 		conversation
> 			`	The Quick Drive hums beneath you, its energy coursing with a strange sense of stability.`

Reviewer: ```suggestion
		require "Quick Drive"
		conversation
			`The Quick Drive hums beneath you, its energy coursing with a strange sense of stability.`
```

			`	As you push the ship to full speed and engage the drive, the void bends smoothly - the instability from before is gone. Yet, as the jump completes, your radar flares to life with multiple unknown contacts.`
			choice
				`	(Scan the ships.)`

> 					goto scan
> 				`	(Prepare weapons-this feels wrong.)`

Reviewer: ```suggestion
				`	(Prepare weapons.)`
```
Don't tell the player what they're thinking.

					goto prepare
			label scan
			`	Your sensors struggle to resolve their signatures. Their vessels are unmarked. Yet there is no mistaking their intent - they are moving to intercept.`
			choice
				`	(Hail them.)`
					goto hail
				`	(Prepare for evasive maneuvers.)`
					goto prepare
			label hail
			`	Your transmission is met with only silence. Then, a single message: "You've awakened the old death. The Tangles will follow."`
			choice
				`	(Attempt to explain.)`

> 					goto explain
> 				`	(This is a trap. Prepare for combat.)`

Reviewer: ```suggestion
				`	(Prepare for combat.)`
```

					goto fight
			label explain
			`	Your words mean nothing. The moment you reply, the ships break formation and move to attack. Whoever they are, they fear the drive you carry - and they mean to destroy you.`
				goto fight
			label prepare
			`	The moment you shift your course, the enemy ships accelerate - there is no doubt now. They are hunting you.`
			label fight
	on complete
		conversation

> 			`	Taaquvaleq awaits as you return, their chromatophores dark with intrigue. "Thou didst return.. yet not unscraped, I see."`

Reviewer: ```suggestion
			`Taaquvaleq awaits as you return, their chromatophores dark with intrigue. "Thou didst return... yet not unscraped, I see."`
```

			`	You relay the attack. At the mention of the enemy's warning, their gaze sharpens.`
			`	"Those who fear the past oft carry its sins. The Quick Drive is thine - the data is all I sought. But know this: if they move against thee once, they may do so again."`
			choice
				`	"Who do you think they were?"`

> 					goto unknown
> 				`	"I can handle myself."`
> 					goto accept
> 			label unknown
> 			`	Taaquvaleq's chromatophores ripple in slow contemplation. "I cannot say. Yet, if they knew of the Isotopes, then they are no strangers to the past."`
> 			label accept
> 			`	"Tread carefully, traveler.", Taaquvaleq gaze lingers on your ship, then back to you-troubled, unreadable, but they say nothing more.`

Reviewer: ```suggestion
				`	"I can handle myself."`
					goto accept
			`	Taaquvaleq's chromatophores ripple in slow contemplation. "I cannot say. Yet, if they knew of the isotopes, then they are no strangers to the past.`
			label accept
			`	"Tread carefully, traveler." Taaquvaleq's gaze lingers on your ship, then back to you - troubled, unreadable. They say nothing more.`
```
